ingest-app/src/cmr/ingest/services/messages.clj

```diff
@@ -47,6 +47,12 @@
    (format "DataCenter short name [%s] and long name [%s] was not a valid keyword combination."
           (:ShortName datacenter) (:LongName datacenter)))
 
+(defn directoryname-not-matches-kms-keywords
+   "Error msg when DirectoryName's ShortName is not in the KMS."
+   [directoryname]
+   (format "DirectoryName short name [%s] was not a valid keyword."
```

Comment:
I thought that would be really confusing. I was gonna say DirectoryName's short name... or Directory name's short name?